src/api/api.ts

```diff
@@ -225,16 +222,13 @@ export class GhostApi {
       debugLog(`Calling ${init.method} ${url}`);
       return fetch(url, {
         ...init,
-        ...{ timeout: getenv.int('GHOST_API_FETCH_TIMEOUT_MS', 10000) },
+        timeout: getenv.int('GHOST_API_FETCH_TIMEOUT_MS', 10000),
```

Comment:
I think that it might be a good idea to define those variables in a separated file.

Then we could do:
`timeout: getenv.int(GHOST_API_FETCH_TIMEOUT_MS, 10000),`

It might be easier to see where variables are used and also we might be able to document them if needed like that:

```typescript
/**
 * some doc
*/
const GHOST_API_FETCH_TIMEOUT_MS = 'GHOST_API_FETCH_TIMEOUT_MS';
```

That would be helpful in case we refactor later for example :+1: